lib/base/Modeler.js

```diff
@@ -40,7 +41,14 @@ export default class CamundaDmnModeler extends DmnModeler {
 
     super({
       ...otherOptions,
-      common,
+      common: {
+        ...common,
+        feelLanguageContext: {
+          parserDialect: 'camunda',
+          ...feelLanguageContext,
```

Comment:
This should probably work:
```suggestion
          ...(common?.feelLanguageContext || {}),
```

Reply:
` ...common.feelLanguageContext,` should be enough as spreading undefined already produces {} and common has a default value set

Reply:
https://github.com/camunda/camunda-dmn-js/compare/773df73..3a743dd6bad6b8a3df4c7a6c7843036db23b1518